botocore/serialize.py

```diff
@@ -486,6 +495,15 @@ def _partition_parameters(self, partitioned, param_name,
             elif isinstance(param_value, bool):
                 partitioned['query_string_kwargs'][
                     key_name] = str(param_value).lower()
+            elif member.type_name == 'timestamp':
+                timestamp_format = member.serialization.get('timestampFormat')
```

Comment:
Any reason you chose not to do this?
```python
timestamp_format = member.serialization.get('timestampFormat', 'iso8601')
```
Dunno if we need a class level variable for this as this seems to be a pretty specific edge case in serialization.

Reply:
Being consistent with everything else in this file.